### PR TITLE
fix: handle NAME command sent without a name

### LINKED ISSUE #75: bug: Using NAME command with no text sets your name to blank
**Overview**

Currently the NAME command operates differently to the others in that it requires a parameter. This isn't too obvious at the moment.

**Acceptance Criteria**

- [ ] Add a reply to "NAME (blank)" that gives better instructions for how to use the command
- [ ] "NAME (blank)" doesn't change your name

**Questions/Assumptions**

- [ ] How does this work with RTL languages?

### PR DESCRIPTION
**Pull request by ghost**
This adds an additional string to be translated, so could do with a review from @kimadactyl before merging.

### CODE REVIEW

**Comment by WheresAlice** [comment]
Lets say the build is deliberately broken until we figure out the best wording for this...

File: application/commands.py [hunk 1]
```diff
@@ -37,10 +37,14 @@ def register(member):
...
-    member.save()
-    response = _("You have set your name to %(name)s") % {'name': params}
-    return response
+    if len(params) > 0:
+        member.name = params
+        member.save()
+        response = _("You have set your name to %(name)s") % {'name': params}
+        return response
+    else:
+        response = _("Please send a name with the NAME command.  For example: 'NAME Alice'")
```
Comment (kimadactyl) [inline application/commands.py:46]:
```python
_("To change your name, send NAME followed by your name")

# Translators: you can generate a random name for a suitable region using https://www.behindthename.com/random/

_("For example: 'NAME  Jabir Reem'")
```

File: features/edit_member_profile.feature [hunk 2]
```diff
@@ -12,3 +12,8 @@ Feature: Members can edit their profile information
     Then My name is "alice"
     And I receive a message containing "alice"
     And My registration time does not change
+
+  Scenario:
+      When I send "NAME" via twilio
+      Then I receive a message containing "Please send a name with the NAME command."
```
Comment (kimadactyl) [inline features/edit_member_profile.feature:18]: Then I receive a message containing "To change your name, send NAME followed by your name"

**Review by kimadactyl — CHANGES_REQUESTED** [review]
Looks great have added a suggested wording!

**Review by kimadactyl — APPROVED** [review]

**Pull request merged**